### PR TITLE
Add playbook to cleanup copr scratch repos

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: obal/data/cleanup_copr.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: 'Get Copr repositories'
+      command: "copr-cli list"
```
Comment (evgeni) [inline obal/data/cleanup_copr.yml]: should this be an explicit `copr-cli list @theforeman`?

File: obal/data/cleanup_copr.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      register: copr_repos
+
+    - set_fact:
+        scratch_repos: "{{ copr_repos.stdout.split('\n') | select('match', '^Name:') | list | replace('Name: ', '') }}"
```
Comment (evgeni) [inline obal/data/cleanup_copr.yml]: the fact should be `copr_scratch_repos`, right?

File: obal/data/cleanup_copr.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    - set_fact:
+        scratch_repos: "{{ copr_scratch_repos | select('match', '.+scratch.+') | list }}"
```
Comment (evgeni) [inline obal/data/cleanup_copr.yml]: the fact should be `copr_scratch_repos`, right?

**Comment by evgeni** [comment]
do we want a `obal cleanup` action for that? or is that very copr specific and you'd use `ansible-playbook` for that?

**Comment by ehelms** [comment]
I added an action and updated the variables. This cleanup is currently copr specific. I added 'cleanup-copr' but I wonder if `copr cleanup` should be the command?

**Comment by evgeni** [comment]
`copr cleanup` is something `obal` would not understand today, as it expects the action to be a single world and all following to be package names. so I'd stick with the current name for now.

`ansible-lint` is unhappy, but LGTM otherwise

**Comment by ehelms** [comment]
Updated to fix the linting errors

**Pull request merged**